pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/util/HelixSetupUtils.java

```diff
@@ -61,6 +62,35 @@
  */
 public class HelixSetupUtils {
 
+  public enum ControllerMode {
```

Comment:
I'll recommend just put the enum name, and use ControllerMode.valueOf(mode.toUpperCase())

Reply:
Done.